src/load-schema.js

```diff
@@ -37,65 +57,59 @@ const loadSchema = async ({ modulePaths, mockMode, useFileSchema, filterSubscrip
       `found duplicates of "${duplicates.join(', ')}"`);
   }
 
-  sources
-    .filter(config => config.dependencies)
-    .forEach(config => config.dependencies
-      .forEach(dependency => {
-        if (!sourceNames.includes(dependency)) {
-          throw new Error(`Cannot load module "${config.name}", ` +
-            `because missing dependency "${dependency}"`);
-        }
-      })
-    );
+  const moduleConfigMap = configs.reduce((prev, curr) => Object.assign(prev, {
```

Comment:
const moduleConfigMap = configs.reduce((prev, cur) => ({...prev, [cur.name]: cur}), {}); ?

Reply:
Same same. Feel free to change it.